scrapers/ManyVids.yml

```
@@ -0,0 +1,148 @@
+  scraper: sceneScraper
+  queryURL: "{url}"
+  queryURLReplace:
+    url:
+      - regex: ^(https:\/\/www\.manyvids\.com\/Video\/)
+        with: https://www.manyvids.com/bff/store/video/
+      - regex: /[^/]*/*$
+        with: ''
+
+sceneByFragment:
+  action: scrapeJson
+  scraper: sceneScraper
+  queryURL: "https://www.manyvids.com/bff/store/video/{filename}"
+  queryURLReplace:
+    filename:
+      - regex: \.(\w+)$
+        with: ''
+
+sceneByURL:
+  - action: scrapeJson
+    url:
+      - https://www.manyvids.com/Video/
+    scraper: sceneScraper
+    queryURL: "{url}"
+    queryURLReplace:
+      url:
+        - regex: ^(https:\/\/www\.manyvids\.com\/Video\/)
+          with: https://www.manyvids.com/bff/store/video/
+        - regex: /[^/]*/*$
+          with: ''
+
+jsonScrapers:
+
+  performerSearch:
+    performer: 
+      Name: models.#.username
+      URL:
+        selector: models.#.url
+        postProcess:
+          - replace:
+                - regex: ^
+                  with: https://www.manyvids.com
+      Image: models.#.avatar.url
+
+  performerScraper:
+    performer:
+      Name: displayName
+      Gender: identification
+      URL: shortLinkUrl
+      Twitter:
+        selector: socLnkTwitter
+        postProcess:
+          - replace:
+                - regex: https://www.twitter.com/
+                  with: ''
+      Instagram: 
+        selector: socLnkInstagram
+        postProcess:
+          - replace:
+                - regex: https://instagram.com/
+                  with: ''
+      Birthdate:
+        selector: dob
+        postProcess:
+          - replace: 
+            - regex: T.*
+              with: ''
+          - parseDate: 2006-01-02
+      Ethnicity: ethnicity
+      Country: location
+      HairColor: hairColor
+      CareerLength:
+        selector: createdAt
+        postProcess:
+          - replace:
+                - regex: '\-.*'
+                  with: ''
+                - regex: '$'
+                  with: ' - Present'
+      Image: portrait
+      Details: description
+
+  sceneSearch:
+    scene:
+      Title: vids.#.title
+      Code: vids.#.id
+      URL: 
+        selector: 'vids.#.{id,slug}.@values'
+        postProcess:
+          - replace:
+            - regex: '\["(\d+)",\s*"([^"]+)"\]'
+              with: $1/$2
+            - regex: ^
+              with: https://www.manyvids.com/Video/
+      Image: vids.#.thumbnail.url
+
+  sceneScraper:
+    scene:
+        Title: data.title
+        Details: data.description
+        Code: data.id
+        URL:
+          selector: data.url
+          postProcess:
+            - replace:
+              - regex: ^
+                with: https://www.manyvids.com
+              - regex: '/*$'
+                with: ''
+        Date:
+          selector: data.launchDate
+          postProcess:
+            - replace: 
+              - regex: T.*
+                with: ''
+            - parseDate: 2006-01-02
+        Image: data.thumbnail
```

Can we use the screenshot instead here? They're the same in every case I've tested with and slightly higher resolution
```suggestion
        Image: data.screenshot
```